data/ka'het/ka'het missions.txt

```diff
@@ -99,56 +120,45 @@ mission "First Contact: Ka'het: Remnant 1B"
 	planet Retilie
-		spaceport `Retilie was used by the builders mainly for fishing, leading to a lack of infrastructural development. The few settlements on the planet were never as advanced as the ones on the nearby worlds, and when the Ka'het arrived there was basically nothing to stop them: after a few attacks, the planet was uninhabited again.`
-		spaceport `	In the oceans, the many species of fish continue to prosper, unaware of the destruction the planet has undergone.`
+		description `	As it was mainly used for fishing, the settlements on this planet were never as advanced as the ones on the nearby worlds, and when the Ka'het arrived there was basically nothing to stop them: after a few attacks, the planet was uninhabited again.`
 	planet Seli'het
-		spaceport `Seli'het was the only Het station the Builders ever completed, orbiting Giribea. The young Ka'heta are brought here once they reach the "Age of Assembly;" strange, lanky machines start molding an exoskeleton onto their bodies right away. As the Ka'het grow, their exoskeleton is expanded and various outfits are added, transforming them into living weapons.`
-		spaceport `	In this system, the Ka'het behave differently than basically anywhere else. They don't seem to ever attack each other, and some even seem to have started collaborating.`
+		description `	This was the only Het station the Builders ever completed, orbiting Giribea. As the young Ka'heta reach their "Age of Assembly," strange, lanky machines start molding an exoskeleton onto their bodies right away. As the Ka'heta grow, their exoskeleton is expanded and various outfits are added, transforming them into living weapons.`
 	planet Siteria
-		spaceport `Siteria is the Builder homeworld, both the place where their journey began and where it ended. After a dozen millennia of underpopulation due to its proximity to the Quarg ring, the Ka'het invasion forced them to retreat. As one of the few remaining planets capable of sustaining farming and by far the one with the best climate, for a few centuries it had again the highest number of inhabitants of all the Builders planets.`
-		spaceport `	The large number of fields and farms around the planet allowed the Builders to survive for centuries, hiding in sparse strongholds and citadels. Almost three hundred years passed before the Ka'het were able to destroy the last colony.`
+		description `	This was the Builder homeworld, both the place where their journey began and where it ended. After a dozen millennia of underpopulation, the Ka'het arrival forced them to retreat. As one of the few remaining planets capable of sustaining farming this was their last stronghold, falling only centuries after the invasion begun.`
```

Comment:
```suggestion
		description `	This was the Builder homeworld, both the place where their journey began and where it ended. After a dozen millennia of underpopulation, the Ka'het arrival forced them to retreat. As one of the few remaining planets capable of sustaining farming, this was their last stronghold, falling centuries after the invasion begun.`
```
`only` implies a shortness to the time elapsed, which seems to contradict `centuries` to me. Even if it is a short period of time in the "grand scheme of things," if this world survives much longer than any other Builder world, then relatively speaking it's not really that "short" a time.